src/main/resources/application.properties

```diff
@@ -30,4 +33,30 @@ spring.datasource.hikari.leak-detection-threshold=15000
 spring.mvc.throw-exception-if-no-handler-found=true
 spring.web.resources.add-mappings=false
 
-sha512.string=5627dcc7bb45a7cf9ddd211168fb727a2005338c5fe1e29faa71fc542bcd59e1bf0c604d65578ba9001984bfa0f25e29a509f130db33c51916bf0ebfbb78b645
+#Logging
+spring.main.banner-mode=off
+logging.pattern.console = ${LOGGING_PATTERN: %green(%d{HH:mm:ss.SSS}) %blue(%-5level) %red([%thread]) %yellow(%logger{15})- %msg%n}
+
+#Swagger doc
+springdoc.api-docs.path=/docs/yml
```

Comment:
@lurldgbodex please confirm that these changes are not already implemented

Reply:
@Am0du  has majorly been doing the configurations with the devops guy. I've tagged him.